test/module.test.js

```diff
@@ -307,8 +307,11 @@ describe('Test sidekick', () => {
         assert.ok(popupOpened === expectedPopupUrl, 'Did not pass additional info in plugin URL');
       }).timeout(IT_DEFAULT_TIMEOUT);
 
-      it.skip('Plugin shows palette', async () => {
+      it('Plugin shows palette', async () => {
         nock.admin(new Setup('blog'));
```

Comment:
```suggestion
```
no admin requests are done. you can remove this

Reply:
Really? But that's wrong, no?